Code/Editor/ViewManager.cpp

```diff
@@ -80,7 +80,8 @@ CViewManager::CViewManager()
     }
     else
     {
-        RegisterQtViewPaneWithName<CRenderViewport>(GetIEditor(), "Perspective", LyViewPane::CategoryViewport, viewportOptions);
+        AZ_Assert(false, "Non-Atom viewport no longer supported");
```

Comment:
Do you mind just removing the `ed_useAtomNativeViewport` CVAR instead here? It should only pop up in a couple places and is already basically vestigial.

Reply:
Addressed in 0a2fa140bf